pytest_splinter/plugin.py

```diff
@@ -70,9 +70,9 @@ def Browser(*args, **kwargs):
     visit_condition = kwargs.pop('visit_condition')
     visit_condition_timeout = kwargs.pop('visit_condition_timeout')
     browser = splinter.Browser(*args, **kwargs)
-    browser.switch_to = browser.driver.switch_to
     browser.wait_for_condition = functools.partial(_wait_for_condition, browser)
     if hasattr(browser, 'driver'):
+        browser.switch_to = browser.driver.switch_to
```

Comment:
implement a simple test which tests only this change, instead of emulating the whole django app

Reply:
Hi @bubenkoff 

You are right, sorry about that! I dropped the Django sample project and mocked the splinter driver to raise AttributeError when hitting browser.switch_to.